### ISSUE TITLE
Testing with Arkanoid

### ISSUE THREAD

**Issue by larsbrinkhoff** [stefanberndtsson/ostis#23]
TL;DR: It works now.

Testing with `make clean ; make ; ostis Arkanoid.st`

a82e7c1bed9b10e7553b9cc5b3f79c71dbd32ac2: Ok
b22044630d894557e1345511ddc52653d91c6a85: 20 bombs
bb61a9f7181cb1355c747866a55e698595f4e2a0: 20 bombs
03105650ad82ce8c0fa0ee8105d7bfc1082018ba: 20 bombs
8912afdc43d0732023cc30edd4e644fa42581f64: 20 bombs
d42e8b8bb462c4f0253a2c2bff9e1c3c1de2687f: 20 bombs
dcb8d87cc8be0213bea58f4b79237c9d7501cff6: 20 bombs
9bf64131bb3b4c176e8928e2b1dc97c5be0d08b9: 20 bombs
3d656994a96340c636d6e1df8c1f25c9c2c785e5: Ok for a while, then hangs with a zillion `DEBUG: We're out of MFP interrupt with ISR still high`
e3a284303290a2a18b9d0c700ff14c49eb120e49: Ok, only occasional `DEBUG: We're out of MFP interrupt with ISR still high`
527c0e94bf32ae4160bc980ce01254f66a9d8326: Segfault
fd36823e6677068e6878a91a70fe2d518bd38889: Strange white border, graphics updates erratically
6e396c7fe7076d7c160c124d024223de48cf8adb: Ok, no `DEBUG` messages

**Comment by larsbrinkhoff** [comment]
Closing for now.
